### PR TITLE
Update selfttest data in dataOut_SM4128_CTR

### LINKED ISSUE #252: The IETF doc uses a different IV in SM4 CTR
Hello, the test data of SM4 is from IEIF (https://tools.ietf.org/html/draft-ribose-cfrg-sm4-04#appendix-A.2). The Iv is inconsistent with libtpms，so do we need to modify the dataOut_SM4128_CTR or MakeIv founction?

### PR DESCRIPTION
**Pull request by JerryDevis**
update the selftest data in dataOut_SM4_CTR
Fix SM4 CTR test data

Resolves #252.

### CODE REVIEW

File: src/tpm2/SymmetricTestData.h [hunk 1]
```diff
@@ -385,16 +385,15 @@ const BYTE dataOut_SM4128_OFB [] = {
...
-    /* FIXME:  The IETF doc uses a different IV than what we do....*/
-    0xAC, 0x32, 0x36, 0xCB, 0x97, 0x0C, 0xC2, 0x07,
-    0x91, 0x36, 0x4C, 0x39, 0x5A, 0x13, 0x42, 0xD1,
-    0xA3, 0xCB, 0xC1, 0x87, 0x8C, 0x6F, 0x30, 0xCD,
-    0x07, 0x4C, 0xCE, 0x38, 0x5C, 0xDD, 0x70, 0xC7,
-    0xF2, 0x34, 0xBC, 0x0E, 0x24, 0xC1, 0x19, 0x80,
-    0xFD, 0x12, 0x86, 0x31, 0x0C, 0xE3, 0x7B, 0x92,
-    0x6E, 0x02, 0xFC, 0xD0, 0xFA, 0xA0, 0xBA, 0xF3,
-    0x8B, 0x29, 0x33, 0x85, 0x1D, 0x82, 0x45, 0x14};
+/* The data are obtained by running the commands as belows:
```
Comment (stefanberger) [inline src/tpm2/SymmetricTestData.h:388]: belows ->below

**Comment by stefanberger** [comment]
Thanks for fixing the typo. Can you now squash the 2nd patch into the first? I will then merge #257 after yours.

**Comment by JerryDevis** [comment]
> Thanks for fixing the typo. Can you now squash the 2nd patch into the first? I will then merge #257 after yours.
I've combined the two patches together，Can this PR be merged in?

**Comment by JerryDevis** [comment]
But, the coverage/coveralls report is always in waiting status.

**Pull request merged**